dao/src/test/resources/logback.xml

```diff
@@ -10,6 +10,9 @@
     <logger name="org.thingsboard.server.dao" level="WARN"/>
     <logger name="org.testcontainers" level="INFO" />
 
+    <!-- Log Hibernate SQL queries -->
+    <logger name="org.hibernate.SQL" level="DEBUG"/>
```

Comment:
Are you sure? This may produce a lot of messages, right?

Reply:
It is only for proof of the output. I  am going to disable this SQL log